src/pages/Auth/SignupPage/hooks/useStep.tsx

```diff
@@ -1,21 +1,28 @@
 /* eslint-disable react/jsx-no-useless-fragment */
+import { useCallback } from 'react';
 import { useNavigate, useParams } from 'react-router-dom';
+import ROUTES from 'static/routes';
 
-function useStep<T extends string>() {
+function useStep<T extends string>(steps: T[]) {
   const navigate = useNavigate();
   const { currentStep } = useParams<{ currentStep: T }>();
 
   if (!currentStep) {
     throw new Error('URL parameter \'currentstep\' is required for signup flow.');
   }
+  const nextStep = useCallback((next: T, options?: { replace: boolean }) => {
+    navigate(`/auth/signup/${next}`, options);
```

Comment:
```suggestion
    navigate(ROUTES.AuthSignup({ currentStep: next, isLink: true }), options);
```
가능하다면 여기도 라우트 상수로 해주면 좋을 것 같아요

Reply:
감사합니다 수정하였습니다!